rekolekcje-engine/src/main/java/pl/oaza/warszawa/dor/rekolekcje/api/security/jwt/JwtUser.java

```diff
@@ -1,13 +1,12 @@
 package pl.oaza.warszawa.dor.rekolekcje.api.security.jwt;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
+import java.util.Collection;
```

Comment:
\[Checkstyle\] WARNING: 'java\.util\.Collection' should be separated from previous import group by one line\.